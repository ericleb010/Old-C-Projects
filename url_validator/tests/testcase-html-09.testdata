-d 1 file:///home/MTA/ealeblanc/asmts/ass4/tests/blah.html
