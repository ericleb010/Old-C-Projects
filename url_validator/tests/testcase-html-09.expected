
Checking file:///home/MTA/ealeblanc/asmts/ass4/tests/blah.html ... OK

>>> Traversing into file:///home/MTA/ealeblanc/asmts/ass4/tests/blah.html:
|	Line 3: checking http://mta.ca ... OK
<<< Done traversing!

